drivers/at86rf2xx/include/at86rf2xx_registers.h

```diff
@@ -50,11 +52,137 @@ extern "C" {
 #define AT86RF2XX_PARTNUM           AT86RF232_PARTNUM
 #elif MODULE_AT86RF233
 #define AT86RF2XX_PARTNUM           AT86RF233_PARTNUM
+#elif MODULE_AT86RFA1
+#define AT86RF2XX_PARTNUM           AT86RFA1_PARTNUM
+#elif MODULE_AT86RFR2
+#define AT86RF2XX_PARTNUM           AT86RFR2_PARTNUM
 #else /* MODULE_AT86RF231 as default device */
 #define AT86RF2XX_PARTNUM           AT86RF231_PARTNUM
 #endif
 /** @} */
 
+/*
+ * memory-mapped transceiver
+ */
+#if defined(MODULE_AT86RFA1) || defined(MODULE_AT86RFR2)
+
+#include <avr/io.h>
+
+/**
+ * @name Register addresses
+ * @{
+ */
+#define AT86RF2XX_REG__TRX_STATUS                               (&TRX_STATUS)
```

Comment:
Why the two underscores between `REG` and `TRX_STATUS`?

Reply:
OK. That is  in my opinion also something to refactor later on. But definitely out of scope of this PR.